src/tests/address.test.ts

```diff
@@ -0,0 +1,7 @@
+import { expect, test } from 'vitest'
+
+test('Should return true', async () => {
```

Comment:
Misschien is het nog handig om hier even een commentje bij te zetten dat dit nu een dummie test is ;)